cg/store/api/status.py

```diff
@@ -533,11 +533,15 @@ def _all_samples_have_sequence_data(links: List[models.FamilySample]) -> bool:
         """Return True if all samples are external or sequenced inhouse."""
         return all((link.sample.sequenced_at or link.sample.is_external) for link in links)
 
-    def analyses_to_upload(self):
+    def analyses_to_upload(self, pipeline: str = None) -> List[models.Analysis]:
         """Fetch analyses that haven't been uploaded."""
         records = self.Analysis.query.filter(
             models.Analysis.completed_at != None, models.Analysis.uploaded_at == None
         )
+
+        if pipeline:
+            records = records.filter(models.Analysis.pipeline.ilike(f"%{pipeline}%"))
```

Comment:
this will currently match MIP-RNA for MIP if there are any Analysis records.. We should prioritize making the pipeline names unique pretty soon

Reply:
Agreed! Let's do that in another PR